metadata/en-US/changelogs/45300.txt

```diff
@@ -0,0 +1,5 @@
+What's Changed:
+* WireGuard obfuscation feature added
```

Comment:
The changelog needs to be replaced. Android changelog is different from the changelog in the release on github. The current android changelog is available from google play.